recipe/meta.yaml

```diff
@@ -20,19 +20,19 @@ requirements:
     - python
     - cross-python_{{ target_platform }}  # [build_platform != target_platform]
     - maturin                             # [build_platform != target_platform]
-    - {{ compiler('c') }}             # [ linux ]
-    - {{ compiler('m2w64_c') }}       # [ win ]
-    - {{ compiler('cxx') }}           # [ linux ]
+    - {{ stdlib("c") }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
     - {{ compiler('rust') }}
+    - {{ compiler('m2w64_c') }}       # [ win ]
+    - {{ stdlib("m2w64_c") }}         # [ win ]
```

Comment:
Try to remove these and rerender. Most Rust recipes work without mingw
```suggestion
```

Reply:
That is what I had a two commits up, it did not work unfortunately.